src/server/schema/object-types/viewer-type.js

```diff
@@ -0,0 +1,17 @@
+import {
+  GraphQLObjectType,
+} from 'graphql';
+import userType from './user-type';
+import {me} from '../../db/controllers/users.controller';
+
+const viewerType = new GraphQLObjectType({
```

Comment:
Maybe a "viewer" is too abstract term? It could mean anything. Something like "currentUserType" would be much more self explanatory.